.mailmap

```diff
@@ -1272,6 +1272,7 @@ Roman Inflianskas <infroma@gmail.com>
 Ronan Lamy <ronan.lamy@gmail.com> <Ronan.Lamy@normalesup.org>
 Rudr Tiwari <rudrtiwari@gmail.com>
 Rupesh Harode <rupeshharode@gmail.com>
+RushabhMehta2005 <mehtarushabh2005@gmail.com> RushabhMehta2005 <139112780+RushabhMehta2005@users.noreply.github.com>
```

Comment:
The first part here is supposed to be your real name as shown in the surrounding entries. The second part should be what is recorded in the commits. If you configure git then you can have your real name recorded in the commits.

Reply:
Okay, I'll make it so that my real name is the first part, rest should be left the same I believe.